image_utils.sh

```diff
@@ -92,14 +100,21 @@ create_partitions() {
   #create bootloader partition
   mkfs.ext2 "${image_loop}p3"
   #create rootfs partition
-  mkfs.ext4 "${image_loop}p4"
+  if [ $is_luks ]; then
+    echo "${PASSWD}" | $CRYPT_PATH luksFormat "${image_loop}p4"
```

Comment:
Running the untrusted cryptsetup binary on the host system is a security problem.